Cargo.toml

```diff
@@ -10,6 +10,7 @@ criterion = "0.2"
 handlebars = { git = "https://github.com/sunng87/handlebars-rust" }
 horrorshow = { git = "https://github.com/Stebalien/horrorshow-rs" }
 liquid = { git = "https://github.com/cobalt-org/liquid-rust" }
+markup = "0.1"
```

Comment:
Please depend on Git master here, as is done for the other crates. This is part of providing a level playing field: all implementations get the performance from their very latest versions.